components/TextFields/src/MDCTextInputControllerLegacyDefault.m

```diff
@@ -1187,6 +1209,36 @@ + (void)setTrailingUnderlineLabelTextColorDefault:
           : MDCTextInputControllerLegacyDefaultInlinePlaceholderTextColorDefault();
 }
 
+- (void)setUnderlineHeightActive:(CGFloat)underlineHeightActive {
+  if (_underlineHeightActive != underlineHeightActive) {
```

Comment:
Nit: Consider using approximate equality here due to floating point imprecision.

```objectivec
if (!MDCCGFloatEqual(_underlineHeightActive, underlineHeightActive)) {
```

It gets tricky depending on the source of the value:
```objectivec
 float someFloat = 0.1f;
  double someDouble = 0.1;
  NSLog(@"%d", someFloat == 0.1); // 0 (false)
  NSLog(@"%d", someFloat == 0.1f); // 1 (true)
  NSLog(@"%d", someDouble == 0.1); // 1 (true)
  NSLog(@"%d", someDouble == 0.1f); // 0 (false)
```

Reply:
Ah, yes. Much smarter.